service/src/main/resources/application.yml

```diff
@@ -132,4 +125,42 @@ drshub:
       mTlsConfig:
         keyPath: rendered/ras-mtls-client.key
         certPath: rendered/ras-mtls-client.crt
+  compactIdHosts:
+    "dg.4503": staging.gen3.biodatacatalyst.nhlbi.nih.gov
+    "dg.712c": staging.gen3.biodatacatalyst.nhlbi.nih.gov
+    "dg.anv0": staging.theanvil.io # this is the old anvil CIB, the new one (drs.anv0) is set explicitly for each env
+    "dg.4dfc": nci-crdc-staging.datacommons.io
+    "dg.f82a1a": gen3staging.kidsfirstdrc.org
+---
+# prod
+spring.config.activate.on-profile: prod
+drshub:
+  compactIdHosts:
+    "dg.4503": gen3.biodatacatalyst.nhlbi.nih.gov # biodatacatalyst
+    # this compact ID intentionally maps to the BDC staging url in prod
+    "dg.712c": staging.gen3.biodatacatalyst.nhlbi.nih.gov # biodatacatalyst
```

Comment:
`dg.712c` is the BDC compact id for preproduction only.
It should never appear or be used in the production tier. 
Cross-deployment-tier data access does not work in general (for Gen3 or TDR)

If this line were deleted, and DRSHub unexpectedly received a DRS URI containing `dg.712c` would this result in the "compact identifier not found" error?
If so, then I think removing this line (or replacing it with a comment) is the appropriate change.